data/caffenet/file_list.txt

```diff
@@ -0,0 +1,3 @@
+/home/sun/prog/smqtk/src/data/caffenet/fish-bike.jpg 0
```

Comment:
These are relative to your home directory. Make these CMake configurable to point to their actual locations in the source tree.